multicluster/controllers/multicluster/commonarea/acnp_resourceimport_controller.go

```diff
@@ -120,10 +120,14 @@ func (r *ResourceImportReconciler) handleResImpDeleteForClusterNetworkPolicy(ctx
 			Name: acnpName,
 		},
 	}
-	if err := r.localClusterClient.Delete(ctx, acnp, &client.DeleteOptions{}); err != nil {
-		return ctrl.Result{}, client.IgnoreNotFound(err)
+	err := client.IgnoreNotFound(r.localClusterClient.Delete(ctx, acnp, &client.DeleteOptions{}))
+	if err != nil {
+		klog.ErrorS(err, "Failed to delete imported ACNP", "acnp", acnpName)
+		return ctrl.Result{}, err
+	} else {
+		r.installedResImports.Delete(*resImp)
+		return ctrl.Result{}, nil
```

Comment:
I think the new writing violates lint rule superfluous-else but not sure why lint didn't fail. 

@luolanzone I think you added that check via https://github.com/antrea-io/antrea/pull/2562?

Reply:
Fixed superfluous-else for this PR

Reply:
@tnqn yes, I will check why lint didn't fail.